applications/structure/bar/tests/weak_damping.output

```diff
@@ -0,0 +1,625 @@
+
+ Time step number = 176     t = 8.75000e+01 -> t + dt = 8.80000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 178     t = 8.85000e+01 -> t + dt = 8.90000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 180     t = 8.95000e+01 -> t + dt = 9.00000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 182     t = 9.05000e+01 -> t + dt = 9.10000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 184     t = 9.15000e+01 -> t + dt = 9.20000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 186     t = 9.25000e+01 -> t + dt = 9.30000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 188     t = 9.35000e+01 -> t + dt = 9.40000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 190     t = 9.45000e+01 -> t + dt = 9.50000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 192     t = 9.55000e+01 -> t + dt = 9.60000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 194     t = 9.65000e+01 -> t + dt = 9.70000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 196     t = 9.75000e+01 -> t + dt = 9.80000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 198     t = 9.85000e+01 -> t + dt = 9.90000e+01
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 200     t = 9.95000e+01 -> t + dt = 1.00000e+02
+________________________________________________________________________________
```

Comment:
Can you adjust the parameters so that this output information is not printed all the time?